charts/portworx/templates/storage-cluster.yaml

```diff
@@ -292,6 +292,9 @@ spec:
     {{- end }}
   {{- end }}
 
+  {{- $isLicenseSecretAdded := ne $licenseSecret "none" }}
+  {{- $shouldRenderEnv := or (ne $envVars "none") .Values.envs $AKSInstall $OKEInstall $isLicenseSecretAdded }}
```

Comment:
Is the following limited to AKS and OKE?

Reply:
Yes some env var are that needs to be included are related to AKS and OKE only